nc/templates/nc/agency_detail.html

```diff
@@ -235,7 +235,7 @@ <h4 class="graph-label">Longitudinal view of annual percent of search by race/et
       <h3>Likelihood of Search by "Stop Cause"</h3>
 
       <p class="help-block">
-        This chart displays the relative statistical likelihood of one
+        This graph displays the relative statistical likelihood of one
         racial/ethnic group being searched as compared to another group for a
```

Comment:
We know how I feel about this line. It is not, in fact, the relative statistical likelihood. That is a thing, and it's not this. This is closer to the "difference in likelihood." relative would be a ratio, but we're subtracting here, so that's a difference.